docker-alt/entrypoint.sh

```
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+USER_ID=${UID:-99}
```

Shellcheck (0.11.0)

[warning] In POSIX sh, UID is undefined.

(SC3028)